guice/src/main/java/com/flipkart/gjex/guice/GuiceBundle.java

```diff
@@ -150,7 +150,7 @@ public void run(T configuration, U configMap, Environment environment) {
 
 		// Add all Grpc Filters to the Grpc Server
 		filters = getInstances(baseInjector, Filter.class);
-		grpcServer.registerFilters(filters, bindableServices);
+		grpcServer.registerFilters(filters, bindableServices, configuration.getGrpc().isEnableAccessLogs());
```

Comment:
lets not pass boolean in arguments. Arguments should be proper class or objects.

Reply:
done